bHY6yh-e06911/cod-yyyiou_bHY6yh-e06911.xml

```diff
@@ -0,0 +1,413 @@
+               <title>Rubrica 6</title>
+               <date when="2025-05-04">May 4, 2025</date>
+            </edition>
+         </editionStmt>
+         <publicationStmt>
+            <authority>SCTA</authority>
+            <availability status="free">
+               <p>Published under a <ref target="https://creativecommons.org/licenses/by-nc-sa/4.0/">Attribution-NonCommercial-ShareAlike 4.0 International (CC BY-NC-SA 4.0)</ref>
+               </p>
+            </availability>
+         </publicationStmt>
+         <sourceDesc>
+            <listWit>
+               <witness xml:id="C" n="cod-yyyiou">Vatican, Biblioteca Apostolica Vaticana, Ross. 429 (IX.119)</witness>
+            </listWit>
+         </sourceDesc>
+      </fileDesc>
+      <encodingDesc>
+         <schemaRef n="lbp-diplomatic-1.0.0" url="https://raw.githubusercontent.com/lombardpress/lombardpress-schema/1.0.0/src/out/diplomatic.rng"/>
+         <editorialDecl>
+            <p>Encoding of this text has followed the recommendations of the LombardPress 1.0.0
+       guidelines for a diplomatic edition.</p>
+         </editorialDecl>
+      </encodingDesc>
+      <revisionDesc status="draft">
+         <listChange>
+            <change when="2025-05-04" status="draft" n="0.0.0">
+               <p>Created file for the first time.</p>
+            </change>
+         </listChange>
+      </revisionDesc>
+   </teiHeader>
+   <text xml:lang="la">
+      <front>
+         <div xml:id="starts-on">
+            <pb ed="#C" n="153-v"/>
+            <cb ed="#C" n="a"/>
+         </div>
+      </front>
+      <body>
+         <div xml:id="bHY6yh-e06911"><!-- p5l3-r6 -->
+            <head xml:id="bHY6yh-e06911-Hd1e108">Rubrica 6</head>
+            <head xml:id="bHY6yh-e06911-Hd1e111" type="question-title">De tristitia</head>
+            <p xml:id="bHY6yh-e06911-d1e114">
+              De tristitia dolo<!--line jumps down--> et merore
+            </p>
+            <p xml:id="bHY6yh-e06911-d1e119">
+              <lb ed="#C" n="9"/><ref xml:id="bHY6yh-e06911-Rd1e122">SAlomon in 
+              <lb ed="#C"/>prouerbis capitulo 12</ref> <quote xml:id="bHY6yh-e06911-Qd1e132">Me<lb ed="#C"/>ror 
+              in corde uiri humiliabit illum</quote> 
+            </p>
+            <p xml:id="bHY6yh-e06911-d1e138">
+              ¶ <ref xml:id="bHY6yh-e06911-Rd1e138">Idem 
+              <lb ed="#C"/>XV capitulo</ref> <quote xml:id="bHY6yh-e06911-Qd1e142">In merore animi deicitur spiritus</quote>
+            </p>
+            <p xml:id="bHY6yh-e06911-d1e147">
+              <lb ed="#C"/>¶ <ref xml:id="bHY6yh-e06911-Rd1e149">Idem 17 capitulo</ref> <quote xml:id="bHY6yh-e06911-Qd1e151">Spiritus tristis exsiccat 
+              <lb ed="#C"/>ossa</quote>
+            </p>
+            <p xml:id="bHY6yh-e06911-d1e155">
+              ¶ <ref xml:id="bHY6yh-e06911-Rd1e157">Idem 25 capitulo</ref> <quote xml:id="bHY6yh-e06911-Qd1e159">Sicud tinea uestimen<lb ed="#C"/>to 
```

Comment:
Can you do the find/replace to add the break='no' where appropriate

For example: 

```suggestion
              ¶ <ref xml:id="bHY6yh-e06911-Rd1e157">Idem 25 capitulo</ref> <quote xml:id="bHY6yh-e06911-Qd1e159">Sicud tinea uestimen<lb ed="#C" break="no"/>to 
```